From d7a33f74d5497ffb932a60611e70d8eabbabe2fe Mon Sep 17 00:00:00 2001
Message-Id: <d7a33f74d5497ffb932a60611e70d8eabbabe2fe.1639776221.git.git@grubix.eu>
From: Michael J Gruber <git@grubix.eu>
Date: Fri, 17 Dec 2021 22:23:36 +0100
Subject: [PATCH] configure: Ignore more options that Fedora spec macros expect

Signed-off-by: Michael J Gruber <git@grubix.eu>
---
 configure | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/configure b/configure
index 6c3a38f1..f574fe54 100755
--- a/configure
+++ b/configure
@@ -308,12 +308,22 @@ for option; do
 	true
     elif [ "${option%%=*}" = '--host' ] ; then
 	true
+    elif [ "${option%%=*}" = '--bindir' ] ; then
+	true
+    elif [ "${option%%=*}" = '--sbindir' ] ; then
+	true
     elif [ "${option%%=*}" = '--datadir' ] ; then
 	true
     elif [ "${option%%=*}" = '--localstatedir' ] ; then
 	true
+    elif [ "${option%%=*}" = '--sharedstatedir' ] ; then
+	true
     elif [ "${option%%=*}" = '--libexecdir' ] ; then
 	true
+    elif [ "${option%%=*}" = '--exec-prefix' ] ; then
+	true
+    elif [ "${option%%=*}" = '--program-prefix' ] ; then
+	true
     elif [ "${option}" = '--disable-maintainer-mode' ] ; then
 	true
     elif [ "${option}" = '--disable-dependency-tracking' ] ; then
-- 
2.34.1.398.ga7b79c7536

